### PR TITLE
feat(table): add tokens and update shim

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/ui/src/shim.cds-core.scss [hunk 1]
```diff
@@ -948,11 +948,28 @@
...
+  --clr-table-cornercellradius: var(--cds-alias-object-border-radius-100);
+  --clr-table-cell-padding-horizontal: var(--cds-global-space-7);
+  --clr-table-cell-padding-vertical: var(--cds-global-space-5);
+  --clr-table-cell-padding: var(--clr-table-cell-padding-vertical) var(--clr-table-cell-padding-horizontal);
+  --clr-table-cell-compact-padding-vertical: var(--cds-global-space-3);
+  --clr-table-cell-compact-padding: var(--clr-table-cell-compact-padding-vertical) var(--clr-table-cell-padding-horizontal);
+  --clr-table-noborder-borderwidth: var(--cds-global-space-0);
+  --clr-table-cell-padding-left: var(--cds-global-space-7); // ATM delete this property
+  --clr-table-noborder-cell-padding-left: var(--cds-global-space-7); // ATM delete this property
+  --clr-table-zero-padding: var(--cds-global-space-0); // ATM delete this property
```
Comment (dtsanevmw) [inline projects/ui/src/shim.cds-core.scss:963]: Are we using this somewhere?

Comment (valentin-mladenov) [reply projects/ui/src/shim.cds-core.scss:963]: `--clr-table-zero-padding` looks like remnant. I'll delete it.

**Review by dtsanevmw — APPROVED** [review]
Looks good! Thanks

File: projects/angular/src/data/_tables.clarity.scss [hunk 2]
```diff
@@ -43,8 +43,7 @@
       table-variables.$clr-table-font-color,
       variables.$clr-use-custom-properties
     );
-    margin: 0;
-    margin-top: variables.$clr_baselineRem_1;
+    margin: variables.$clr_baselineRem_1 0 0 0;
```
Comment (dtsanevmw) [inline projects/angular/src/data/_tables.clarity.scss:46]: Think about exposing this as a property in `_properties.tables.scss` although we don't need to change it in the shim.

Comment (valentin-mladenov) [reply projects/angular/src/data/_tables.clarity.scss]: corrected in commit 69bb71b.

**Pull request merged**